telepathy-glib-uniquify.patch

```diff
@@ -0,0 +1,24 @@
+From fd733c82ba71201ab7793a38e8fc2fb47a9062f9 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
+Date: Sun, 19 Feb 2017 00:40:05 +0100
+Subject: [PATCH] base-client: Append a single element when uniquifying name
+
+---
+ telepathy-glib/base-client.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/telepathy-glib/base-client.c b/telepathy-glib/base-client.c
+index 600f292..7f5c155 100644
+--- a/telepathy-glib/base-client.c
++++ b/telepathy-glib/base-client.c
+@@ -1347,7 +1347,7 @@ tp_base_client_constructed (GObject *object)
+       unique = tp_escape_as_identifier (tp_dbus_daemon_get_unique_name (
+             self->priv->dbus));
+
+-      g_string_append_printf (string, ".%s.n%u", unique, unique_counter++);
++      g_string_append_printf (string, ".%sn%u", unique, unique_counter++);
+       g_free (unique);
+     }
+
+-- 
+2.9.3
```

Comment:
re. the `OSError: [Errno 38] Function not implemented: '/run/user/1000/doc/c9878c30/chart.png'` error, I remember seeing that before.. but it went away after flatpak/runtimes updates...

Reply:
Hmm, probably an issue not yet solved for arch